packages/blocks/README.md

```diff
@@ -379,15 +379,15 @@ _Parameters_
 
 _Returns_
 
--   `(Array<WPBlockVariation>|void)`: Block variations.
+-   `(WPBlockVariation[]|void)`: Block variations.
```

Comment:
In this case, the tuples still get the parens :thinking:

Reply:
This is another example of the parser just copying the way the type is annotated out of the curly braces: https://github.com/WordPress/gutenberg/blob/1f5af853b83999f773d1e10ed250f06a294c73df/packages/blocks/src/api/registration.js#L537